<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" rowspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Vector 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Vector&lt;long long&gt; №1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>150</number>
          </attribute>
          <column>
           <property name="text">
            <string>Значение</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Адрес</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Vector 2</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Vector&lt;long long&gt; №2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>150</number>
          </attribute>
          <column>
           <property name="text">
            <string>Значение</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Адрес</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Возврат значений</string>
    </property>
    <addaction name="action_capacity"/>
    <addaction name="action_size"/>
    <addaction name="action_empty"/>
    <addaction name="action_maxSize"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Вставка</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_insert"/>
    <addaction name="action_emplace"/>
    <addaction name="action_12"/>
    <addaction name="action_"/>
    <addaction name="separator"/>
    <addaction name="action_erase"/>
    <addaction name="action_clear"/>
    <addaction name="action_13"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Копия</string>
    </property>
    <addaction name="action_assign"/>
    <addaction name="action_swap"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Вставка</string>
   </property>
  </action>
  <action name="action_insert">
   <property name="text">
    <string>Вставка элементов</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Вставка в конец</string>
   </property>
  </action>
  <action name="action_emplace">
   <property name="text">
    <string>Вставка п.п. конструктора</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Вставка п.п. конструктора в конец</string>
   </property>
  </action>
  <action name="action_">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Удаление</string>
   </property>
  </action>
  <action name="action_erase">
   <property name="text">
    <string>Удаление элементов</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_12">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="action_13">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="action_constrDef">
   <property name="text">
    <string>Размер + элемент для копии</string>
   </property>
  </action>
  <action name="action_constrinList">
   <property name="text">
    <string>Элементы</string>
   </property>
  </action>
  <action name="action_clear">
   <property name="text">
    <string>Очистка</string>
   </property>
  </action>
  <action name="action_assign">
   <property name="text">
    <string>Сплагиатить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_swap">
   <property name="text">
    <string>Обмен</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_capacity">
   <property name="text">
    <string>Объем памяти</string>
   </property>
  </action>
  <action name="action_size">
   <property name="text">
    <string>Размер</string>
   </property>
  </action>
  <action name="action_empty">
   <property name="text">
    <string>Пустой ли</string>
   </property>
  </action>
  <action name="action_maxSize">
   <property name="text">
    <string>Максимальный размер</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
